lib/client-assets.php

```diff
@@ -234,6 +234,14 @@ function gutenberg_register_vendor_scripts( $scripts ) {
 		'4.17.19',
 		true
 	);
+
+	gutenberg_register_vendor_script(
+		$scripts,
+		'object-fit-polyfill',
+		'https://unpkg.com/objectFitPolyfill@2.3.0/dist/objectFitPolyfill.min.js',
```

Comment:
We should not be using CDNs like unpkg.
Historically WordPress bundles whatever it needs, but in this case I'm not sure what would be best to do. This is a polyfill for IE only since other browsers natively support object-fit, so perhaps this should be left out?

Reply:
Yeah, I wasn't 100% certain the best way to go about it.

@aduth might have some insight. Looks like he's done some things with polyfills.

The file doesn't actually get loaded from the CDN in the bundle. [get-vendor-scripts.php](https://github.com/WordPress/gutenberg/blob/master/bin/get-vendor-scripts.php) downloads them for the bundle.

And the way this is set up, the script will only be loaded in browsers that fail the test ([`"objectFit" in document.documentElement.style`](https://github.com/WordPress/gutenberg/pull/25171/files/44a55815352eeab85becb5608a2f23f3c4a13de5#diff-620130744b6b73b822a3df609671d930R650)). I confirmed that it loads in IE, but not in Firefox or Chrome.

An alternative might be to just use the `transform: translate(-50%, -50%); top: 50%; left: 50%` trick; although, that would break `object-position` in IE.

Reply:
> The file doesn't actually get loaded from the CDN in the bundle. get-vendor-scripts.php downloads them for the bundle.

I didn't know that, pretty cool! :+1: 

To be honest, IE users can't (and don't) expect things to work the same way they'd work on other browsers. The fact that WP still supports IE11 doesn't necessarily mean that things should be identical. As long as IE users can see and consume the content, some visual inconsistencies are to be expected. That is why global-styles use CSS variables too...

Reply:
@aristath Was removing the polyfill a blocker for getting this PR approved? Or are we waiting for @aduth's opinion for this?

Reply:
@noisysocks was this polyfill backported to Core somehow?

Reply:
Nevermind, I just found it :)